docker-compose.yaml

```
@@ -0,0 +1,25 @@
+version: "3"
+services:
+  proxy:
+    image: nginx:1.17.4
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    ports:
+      - ${PRISM_PROXY_PORT}:8080
```

8080 is just fine. Again this is an example we're going to put in the documentation, not a real feature
```suggestion
      - 8080:8080
```